aos/aos_milestones/IMpg/check_impg_run.ipynb

```diff
@@ -0,0 +1,798 @@
+{
```

Comment:
<p>Same comment on the units, like the above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_analysis_notebooks/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>done</p><p> </p>